troubleshooting/general-issues.md

```diff
@@ -59,7 +59,7 @@ There are several common cases when such a mismatch occurs:
 
 * The position and size of the `<app>` element (or however the root component of your Blazor app is called) does not match the `<body>`.
 
-* There is more than one `<TelerikRootComponent>` in the app (for example, a certain Razor Component has its own) - there should be only one instance in the main layout.
+* There is more than one `<TelerikRootComponent>` in the app (for example, a certain Razor Component has its own). There should be only one instance - either in `MainLayout`, or in a separate layout file, which is referenced in `MainLayout`.
```

Comment:
Not sure if this addition will make it clearer, or will add confusion. Early versions had similar long-winded explanations that gave people options, and that caused questions and concerns like "which is better" or "which should I do".